site/components/src/main/java/uk/nhs/digital/common/feed/RssModifier.java

```diff
@@ -15,7 +15,7 @@
 import org.hippoecm.hst.core.request.HstRequestContext;
 import org.hippoecm.repository.util.DateTools;
 import org.jdom2.Element;
-import org.jdom2.Namespace;
+//import org.jdom2.Namespace;
```

Comment:
Please remove the commented import statement

Reply:
Of course, not a problem. I was not too sure who commented it out and that maybe there was a reason for it, so did not want to remove it.